styles/colors-silicon-day.less

```diff
@@ -300,17 +301,17 @@
 
 // Toggle Colors
 // ---------------------------------------
-@agate-togglebutton-indicator-off-bg-color: black;
-@agate-togglebutton-indicator-on-bg-color: @agate-highlight;
-@agate-togglebutton-indicator-off-border-color: @agate-button-border-color;
-@agate-togglebutton-indicator-on-border-color: @agate-accent;
+@agate-togglebutton-indicator-off-bg-color: @silicon-dark-gray;
+@agate-togglebutton-indicator-on-bg-color: @silicon-accent;
```

Comment:
I think this one perhaps should have been `@agate-accent`.

It may not be good to have variables so close in name with very different functionality.

Reply:
i don't know how I missed that one. changed it to agate-accent